cargo-concordium/src/templates/cis2-nft-cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "cis2_wccd"
+version = "0.1.0"
+authors = ["Concordium <developers@concordium.com>"]
```

Comment:
Ideally this would not be Concordium, but rather generated from the users' .gitconfig or similar environment variable, or supplied on the command-line.